packages/backend/src/modules/attachment/attachment.repository.ts

```diff
@@ -0,0 +1,40 @@
+import fsPromises from 'fs/promises'
+import { CreateAttachmentDTO, AttachmentDTO } from 'shared'
+import { Attachment } from './Attachment'
+import { fileNameHash } from '../../helpers/fileNameHash'
+import { Repository } from '../../application/Repository'
+import AttachmentModel from './attachments.model'
+
+export class AttachmentRepository implements Repository<AttachmentDTO> {
```

Comment:
DTO should be never stored in repository. Why you created mapper if at the end of the day the dto is stored?

Reply:
please add correct methods to this Repository interface (save/findById/deleteById)

Reply:
fix it